pkg/app/configure.go

```diff
@@ -110,9 +117,27 @@ func getReader(location string) (io.Reader, error) {
 		if err != nil {
 			return nil, fmt.Errorf("parsing location as URL %s: %w", location, err)
 		}
-		resp, err := http.Get(url.String()) //nolint
+		client := &http.Client{}
+		req, err := http.NewRequestWithContext(ctx, "GET", url.String(), nil)
```

Comment:
ll we create another issue to replace the direct usage of 'net/http' with our mock friendly version?

Reply:
This package here is it? 
https://github.com/fidelity/kconnect/tree/main/pkg/http

Reply:
Thats it. There is already an instance on the App https://github.com/fidelity/kconnect/blob/main/pkg/app/app.go#L49